INSTALL.md

```diff
@@ -15,50 +32,51 @@ Update WSL's software before continuing. If you chose Debian, Ubuntu, or another
 
-For example, if you want to store pokered in **C:\Users\\*\<user>*\Desktop**, enter this command:
+For example, if you want to store **pokered** in `C:\Users\<user>\Desktop`, enter this command:
 
 ```bash
 cd /mnt/c/Users/<user>/Desktop
 ```
 
-(The Windows `C:\` drive is called `/mnt/c/` in WSL. Replace *\<user>* in the example path with your username.)
+(The Windows `C:\` drive is called `/mnt/c/` in WSL. Replace `<user>` in the example path with your username.)
 
 If this works, then follow [the instructions for **Linux**](#linux) below for whatever distribution you installed for WSL.
 
-Otherwise, continue reading below for [the older Windows instructions](#windows).
+Otherwise, continue reading below for [the older Windows instructions](#windows-alternate).
+
 
+## Windows (alternate)
 
-## Windows
+*These instructions are the only option for older versions of Windows, but they also work for Windows 10 or 11 if you prefer not to use the [above](#windows-10-or-11).*
 
-Download [**Cygwin**](http://cygwin.com/install.html): **setup-x86_64.exe** for 64-bit Windows, **setup-x86.exe** for 32-bit.
+Download [**Cygwin**](http://cygwin.com/install.html): use the installer **setup-x86_64.exe** for 64-bit Windows, **setup-x86.exe** for 32-bit.
 
 Run setup and leave the default settings. At the "**Select Packages**" step, choose to install the following, all of which are in the "**Devel**" category:
 
 - `make`
 - `git`
 - `gcc-core`
 
-Double click on the text that says "**Skip**" next to each package to select the most recent version to install.
+Double click on the text that says "**Skip**" next to each package to select the most recent version to install. Click "**Next**" and go through the remaining installer screens until it's finished.
 
-Then follow the [**rgbds** install instructions](https://rgbds.gbdev.io/install#pre-built) for Windows with Cygwin to install **rgbds 0.5.2**.
+Once you have finished installing Cygwin, follow the [**rgbds** install instructions](https://rgbds.gbdev.io/install#pre-built) (click **Windows**, then **Cygwin, MSYS2**) to install **rgbds 0.5.2**.
 
-**Note:** If you already have an older rgbds, you will need to update to 0.5.2. Ignore this if you have never installed rgbds before. If a version newer than 0.5.2 does not work, try downloading 0.5.2.
+***Note** (ignore this if you have never installed **rgbds** before):  
+If you already have an older version of **rgbds**, you will need to update to **0.5.2**. If a version newer than **0.5.2** does not work, try downloading **0.5.2.***
 
-Now open the **Cygwin terminal** and enter the following commands.
+Now open the **Cygwin terminal** and enter the following commands. You can open the terminal by double-clicking `Cygwin.bat`, which is in ``C:\cygwin64\``.
```

Comment:
[StackOverflow ](https://stackoverflow.com/questions/34602584/how-can-i-open-a-cygwin-terminal) agrees with you regarding your choice of terminal, but for new users "open the Cygwin terminal" isn't a self-evident instruction.

If you use mintty as your terminal to launch the bash shell, you have to manually login to bash, whereas cmd via `cygwin.bat` handles that for you. It's a difference of two commands, which is not a lot in the grand scheme of things, but the audience here is specifically users in need of help, so I think it's justifiable to provide instructions that have fewer steps to get to the same place.

Assuming you plan to clone pokered into `home/<user>`, these are the commands needed to build pokered using mintty:

```
/bin/bash --login -i
cd /home/<user>
git clone https://github.com/pret/pokered
cd pokered
make
```

Whereas cmd via `cygwin.bat` only requires the latter 3 commands (which are currently included in the documentation).

Would it be excessive to document both methods of getting there under discussion so that all paths converge on the existing documented commands?

Reply:
Have you had a chance to consider the options re: instructing users how to open the cygwin terminal?

Reply:
It's been a long time since I've done a fresh cygwin install. Does the installer not simply put a mintty shortcut on your desktop?

Reply:
It does not, no. (At least, it didn't for me.)

Reply:
It's an option on the last page of the installer:

![image](https://user-images.githubusercontent.com/8334194/183303033-76aac074-cecf-4f59-afee-2e1e84acfd56.png)

(I'm not sure if it's checked by default and unchecked here since I'm re-running the installer to update packages, or if it's unchecked by default now)

Reply:
Does the desktop shortcut go to mintty?

Reply:
Yes; the shortcut is `C:\cygwin64\bin\mintty.exe -i /Cygwin-Terminal.ico -`.

Reply:
I should note it's enabled by default, only disabled if there already is one.